package.json

```diff
@@ -10,14 +10,15 @@
   },
   "author": "articulate",
   "license": "MIT",
-  "dependencies": {},
+  "dependencies": {
+    "ioredis": "^4.17.3",
```

Comment:
ioredis is just a dev dependency as the library doesn't create a client, it just takes a client. However maybe it should be a [peerDependency](https://docs.npmjs.com/files/package.json#peerdependencies)?